vllm/model_executor/layers/fused_moe/modular_kernel.py

```diff
@@ -795,4 +874,7 @@ def forward(
             self.fused_experts.finalize_weight_and_reduce_impl(),
         )
 
-        return output
+        if self.shared_experts is None:
+            return output
+        else:
```

Comment:
Nit: `assert shared_output is not None`?

Reply:
`shared_output` is declared to be a `torch.Tensor` so it should never be `None`